conf/app.routes

```diff
@@ -12,16 +12,16 @@ GET     /arn/:arn/team-member/:userId/groups    uk.gov.hmrc.agentpermissions.con
 
 # Custom access groups
-POST    /arn/:arn/groups                 uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.createGroup(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn)
-GET     /arn/:arn/groups                 uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.groups(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn)
-GET     /groups/:gid                     uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.getGroup(gid: String)
-GET     /custom-group/:gid               uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.getCustomGroupSummary(gid: String)
-DELETE  /groups/:gid                     uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.deleteGroup(gid: String)
-PATCH   /groups/:gid                     uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.updateGroup(gid: String)
-PUT     /groups/:gid/add-unassigned      uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.addUnassignedMembers(gid: String)
-PATCH   /groups/:id/members/add          uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.addTeamMemberToGroup(id: String)
-
-GET     /group/:gid/clients              uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.getPaginatedClientsForGroup(gid: String, page: Int, pageSize: Int, search: Option[String], filter: Option[String])
+POST    /arn/:arn/groups                    uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.createGroup(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn)
+GET     /arn/:arn/groups                    uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.groups(arn: uk.gov.hmrc.agentmtdidentifiers.model.Arn)
+GET     /groups/:gid                        uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.getGroup(gid: String)
+GET     /custom-group/:gid                  uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.getCustomGroupSummary(gid: String)
+DELETE  /groups/:gid                        uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.deleteGroup(gid: String)
+PATCH   /groups/:gid                        uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.updateGroup(gid: String)
+PUT     /groups/:gid/add-unassigned         uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.addUnassignedMembers(gid: String)
+PATCH   /groups/:id/members/add             uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.addTeamMemberToGroup(id: String)
+GET     /group/:gid/clients                 uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.getPaginatedClientsForGroup(gid: String, page: Int, pageSize: Int, search: Option[String], filter: Option[String])
+DELETE  /groups/:gid/client/:clientId       uk.gov.hmrc.agentpermissions.controllers.AccessGroupsController.removeClient(gid: String, clientId: String)
```

Comment:
are we going with `group` or `groups` ? Just noticed the GET endpoint above is the odd one out, should update it later